erizo_controller/erizoJS/erizoJS.js

```diff
@@ -79,7 +81,10 @@ var log = logger.getLogger('ErizoJS');
 var threadPool = new addon.ThreadPool(GLOBAL.config.erizo.numWorkers);
 threadPool.start();
 
-var ejsController = controller.ErizoJSController(threadPool);
+var ioThreadPool = new addon.IOThreadPool(GLOBAL.config.erizo.numIOWorkers);
+ioThreadPool.start();
```

Comment:
We could potentially not start the `ioThreadPool` if we're not using nicer right?